### DISCUSSION TITLE
[How To] Update ReadDeck

### DISCUSSION THREAD

**Discussion by Node815** [tteck/Proxmox#2448] [General]
I don't think this fits in show and tell, but there was an update to 0.12 so I looked at their site.  It's pretty simple, you just have to login to the lxc container, and do:

`wget https://codeberg.org/readeck/readeck/releases/download/0.12.0/readeck-0.12.0-linux-amd64`

It will download, then you rename it with: 
```
mv readeck-0.12.0-linux-amd64  readeck
chmod +x readeck 
systemctl stop readeck 
```
Next, you can use Midnight commander to Move it over to the /opt/readeck directory or do the below commands:

```
rm /opt/readeck/readeck 
mv redeck  /opt/readeck/
```
Finally, 

`systemctl start readeck`

This is how I updated mine. I hope this works for anyone else.

**Comment by tteck** [comment]
You can now type `update` in the console.
